numpy/core/tests/test_scalarmath.py

```diff
@@ -419,6 +420,21 @@ def test_clongdouble___int__(self):
             assert_raises(OverflowError, x.__int__)
             assert_equal(len(sup.log), 1)
 
+    @dec.skipif(np.finfo(np.double) == np.finfo(np.longdouble))
```

Comment:
This test fails on ppc64. 

I think I'm going to add `@dec.skipif(platform.machine().startswith("ppc64"))` here.